driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientTransactionalGraph.java

```diff
@@ -0,0 +1,74 @@
+package org.apache.tinkerpop.gremlin.orientdb;
+
+import org.apache.commons.configuration.Configuration;
+import org.apache.tinkerpop.gremlin.structure.Transaction;
+
+public class OrientTransactionalGraph extends OrientGraph {
```

Comment:
Mh, nope. But you are right. Just implementing tx in OrientGraph also works just fine. I'll rebase the commit and update the PR after some testing.